src/vm/tieredcompilation.cpp

```diff
@@ -367,11 +371,15 @@ BOOL TieredCompilationManager::CompileCodeVersion(NativeCodeVersion nativeCodeVe
     EX_TRY
     {
         pCode = pMethod->PrepareCode(nativeCodeVersion);
+        LOG((LF_TIEREDCOMPILATION, LL_INFO10000, "TieredCompilationManager::CompileCodeVersion Method=0x%x (%s::%s), code version id=0x%x, code ptr=0x%x\n",
```

Comment:
The format character for the pMethodDesc is incorrect (%x, should be %p and probably formatted as %pM like in the preexisting log call below)